helm_deploy/licences/values.yaml

```diff
@@ -3,3 +3,6 @@
 
 generic-prometheus-alerts:
   targetApplication: licences
+
+  ingress:
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```

Comment:
ingress needs to be top level and this needs to be in an annotations section

Reply:
fixed